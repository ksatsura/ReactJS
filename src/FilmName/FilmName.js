//imports
import React from 'react';
import '../style.css';

const FilmName = (props) => {

    return <p className='film-asset-name'>{props.title}</p>;
};

export default FilmName;