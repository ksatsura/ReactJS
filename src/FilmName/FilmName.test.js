import React from 'react';
import FilmName from './FilmName.js';
import Adapter from 'enzyme-adapter-react-16';
import {shallow} from 'enzyme';

describe('FilmName component', () => {

    let title;

    beforeAll(() => {
        title= '';
    });

    it('should be render correctly', () => {
        const component = shallow(<FilmName title={title} />);
        
        expect(component).toMatchSnapshot();
    });
});