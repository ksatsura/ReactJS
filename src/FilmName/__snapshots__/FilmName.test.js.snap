// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmName component should be render correctly 1`] = `ShallowWrapper {}`;
